js/backend.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+(function () {
+  var URLGET = 'https://javascript.pages.academy/code-and-magick/data';
+  var URLPOST = ' https://javascript.pages.academy/code-and-magick';
```

Comment:
Лучше бы в одну константу

Reply:
```suggestion
  var API_URL = 'https://javascript.pages.academy/code-and-magick';
```

А уже в методе добавлять "/data" по необходимости